apps/web/src/components/convex-boundary.tsx

```
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { useConvex } from 'convex/react';
+
+interface ConvexBoundaryProps {
+  children: React.ReactNode;
+  fallback?: React.ReactNode;
+}
+
+export function ConvexBoundary({ children, fallback }: ConvexBoundaryProps) {
+  // Try to access the Convex context
+  try {
+    const convex = useConvex();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)